documentation/docs/tutorials/validate_university_degree.mdx

```diff
@@ -52,12 +53,12 @@ As described in the [Digital Identities Solution](https://www.iota.org/solutions
 ## Key Storage
 
 - In this tutorial, you will store the key pairs for every newly created or updated DID document in Weakhold.
-- Ok, ok, it’s just a couple of JSON files in a folder, but it gets the job done.
-- The files are stored in the [Weakhold Folder](https://github.com/adrian-grassl/iota-identity-tutorial/tree/master/weakhold) (e.g. ./weakhold/Alice.json).
+- Okay, okay, it is just a couple of JSON files in a folder, but it gets the job done (TW Note: remove or embrace the brevity?).
```

Comment:
The next bullet point sort of covers this, so I think you could remove it.

Reply:
This bullet point has a really weird change in tone IMO.